src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -86,6 +86,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Private ReadOnly Property IClauses As ImmutableArray(Of ICaseClause) Implements ICase.Clauses
             Get
+                ' `CaseElseClauseSyntax` is bound to `BoundCaseStatement` with an empty list of case clauses, 
+                ' so we explicitly create an IOperation node for Case-Else clause to differentiate it from Case clause.
                 If Me.CaseStatement.CaseClauses.IsEmpty Then
```

Comment:
Can this be true for erroneous Case clause? We should probably make this check stronger, i.e. make sure the syntax is in fact Case Else.